src/library_noderawfs.js

```diff
@@ -5,7 +5,7 @@
 
 mergeInto(LibraryManager.library, {
   $NODERAWFS__deps: ['$ERRNO_CODES', '$FS', '$NODEFS'],
-  $NODERAWFS__postset: 'if (ENVIRONMENT_IS_NODE) {' +
+  $NODERAWFS__postset: 'if (ENVIRONMENT_IS_NODE || ENVIRONMENT_HAS_NODE) {' +
```

Comment:
https://github.com/emscripten-core/emscripten/blob/5e3029d055d2ee55d486d602c82f04feaf3e6c2c/src/shell.js#L78-L79

Per the code `ENVIRONMENT_IS_NODE` includes `ENVIRONMENT_HAS_NODE`. I think `ENVIRONMENT_HAS_NODE` alone should suffice given that the intention here is to allow all Node-like environments to have NODERAWFS.

Reply:
Thanks. I've made the change.